test/Datadog.Trace.ClrProfiler.IntegrationTests/AspNetCore/AspNetCoreMvc31Tests.cs

```diff
@@ -0,0 +1,25 @@
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Datadog.Trace.ClrProfiler.IntegrationTests.AspNetCore
+{
+    public class AspNetCoreMvc31Tests : AspNetCoreMvcTestBase
+    {
+        public AspNetCoreMvc31Tests(ITestOutputHelper output)
+            : base("AspNetCoreMvc31", output)
+        {
+            // EnableDebugMode();
+        }
+
+#if NETCOREAPP3_1
+        [Fact]
```

Comment:
This could be a good use of the `TargetFrameworkVersionsFact` where we hard-code what target frameworks this will be run on (and skipped for others). I'm not sure what the consensus is if we like the compiler directive or the "Fact" better for determining what tests are compiled/run

Reply:
I forgot we had that, thanks!

Reply:
Oh, this fails to build without the compiler directive because `PackageVersions.AspNetCoreMvc2` in the following line only exists for `netcoreapp2.1`:
```csharp
[MemberData(nameof(PackageVersions.AspNetCoreMvc2), MemberType = typeof(PackageVersions))]
```

Reply:
In the end, to build `Samples.AspNetCoreMvc2` correctly, I had to replace the `PackageReference` to `Microsoft.AspNetCore` and `Microsoft.AspNetCore.Mvc` with `Microsoft.AspNetCore.App` (without a specific version), so I removed `PackageVersions.AspNetCoreMvc2` entirely.

Reply:
I'm resolving this comment since the preprocessor directives are gone now.